examples/plot_mdm_spd_helper.py

```diff
@@ -0,0 +1,145 @@
+import numpy
+import matplotlib
```

Comment:
We can organize this code, by putting these helpers functions in the code codebase.

They could go:
- in `learning/mdm.py` as auxiliary functions, 
- in `geometry/spd_matrices.py`, 
- in `geomstats/visualization.py`, 
- maybe in the folder `geomstats/datasets/`,
- if you want to use real data (not too memory consuming so that we can store them on github), you can put them hin `examples/data`.

Reply:
I reorganized the code into the modules for SPD matrices, rotation matrices, visualization and datasets. The MDM module itself remains unchanged.